### PR TITLE
Book download into a new collection for editing (BL-13035)

### PR DESCRIPTION
**Pull request by JohnThomson**
Requires a parallel change on Bloom Desktop, not yet implemented; but it should be harmless to merge this independently.

- - -
This change is [Reviewable](https://reviewable.io/reviews/BloomBooks/BloomLibrary2/535)

### CODE REVIEW

**Review by andrew-polk — CHANGES_REQUESTED** [review]
> Reviewed 5 of 5 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomLibrary2/535)* status: all files reviewed, 1 unresolved discussion (waiting on @JohnThomson)

___
*[`src/components/BookDetail/BookDetail.tsx` line 389 at r1](https://reviewable.io/reviews/BloomBooks/BloomLibrary2/535#-NpvO7PY53vKx9M7amYO:-NpvO7PY53vKx9M7amYP:b-1s2254) ([raw file](https://github.com/BloomBooks/BloomLibrary2/blob/1d29f957faa855fee42cc63559425a808c93c192/src/components/BookDetail/BookDetail.tsx#L389)):*
> ```XML
>                         )}
> 
>                         <BookExtraPanels book={props.book} />
> ```

Did we lose BookExtraPanels?

**Comment by JohnThomson** [comment]
This is ready for a final review. Converting to draft because we don't want to merge until 5.7 is in beta, since otherwise it refers people to a download page that doesn't offer the version they need.

**Review by andrew-polk — COMMENTED** [review]
Looks good.

Hm. Since you already made some of the UI changes which will come with other cards, should we just hide the new bit and merge this?

In other words, the other cards will be able to build on this. Otherwise, there will be duplication of some of these changes.

> Reviewed 5 of 5 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomLibrary2/535#-:-NpzJim38ttlLKALWgpA:bnnoqoz)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @JohnThomson)

**Review by JohnThomson — COMMENTED** [review]
Done

> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomLibrary2/535#-:-Nq-aMZ4Ew2LkPxT6DTC:bb6un1j)* status: 5 of 7 files reviewed, all discussions resolved (waiting on @andrew-polk)

**Review by andrew-polk — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomLibrary2/535)* status: all files reviewed, 1 unresolved discussion (waiting on @JohnThomson)

___
*[`src/connection/ParseServerConnection.ts` line 45 at r3](https://reviewable.io/reviews/BloomBooks/BloomLibrary2/535#-Nq-dly61J2sdyYREpBv:-Nq-dly61J2sdyYREpBw:b-yoccl3) ([raw file](https://github.com/BloomBooks/BloomLibrary2/blob/73565a04b5a0484b9c30b572b9b7c940b386b905/src/connection/ParseServerConnection.ts#L45)):*
> ```XML
> 
> export function getConnection(): IConnection {
>     return dev; // TODO: revert this!
> ```

Oops. : )

File: src/connection/ParseServerConnection.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ const local: IConnection = {
 };
 
 export function getConnection(): IConnection {
+    return dev; // TODO: revert this!
```
Comment (andrew-polk) [inline src/connection/ParseServerConnection.ts:45]:
```suggestion

```

**Pull request merged**